system_test/helpers/aest_nodes.erl

```diff
@@ -561,14 +561,14 @@ mgr_prepare_specs(NodeSpecs, State) ->
         #{Mod := BackendState} = Backends,
         Acc#{Name => Mod:peer_from_spec(S, BackendState)}
     end, CurrAddrs, PrepSpecs),
-    lists:map(fun(#{peers := Peers} = Spec) ->
+    lists:map(fun(Spec) ->
         NewPeers = lists:map(fun
             (Addr) when is_binary(Addr) -> Addr;
             (Name) when is_atom(Name) ->
                 case maps:find(Name, AllAddrs) of
                     {ok, Addr} -> Addr;
                     _ -> error({peer_not_found, Name})
                 end
-        end, Peers),
-        Spec#{peers := NewPeers}
+        end, maps:get(peers, Spec, [])),
```

Comment:
Shall spec in [aest_nodes:node_spec](https://github.com/aeternity/epoch/blob/1134b14ba9eacbb68e59ca4948d50ab4af8abb5b/system_test/helpers/aest_nodes.erl#L71) be updated then?

Reply:
I guess it's not needed, a spec of `[Thing]` can be an empty list AFAIK